.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,58 @@
+**The following are 'hard suggestions': if you have the information
+available to you, you really should put it in the report. Choose one of
+the following templates from below that best fits the issue you wish to
+report.**
+
+# Performance issue
+___This section should be used for any report regarding performance issues___
+
+###  Timings or Profile link:
+___We ask that all timings/profiles are a link, not a screenshot. Screenshots inhibit our ability to figure out the real
+cause of the issue.___
+
+### Description of issue:
+___If applicable___
+
+### Plugin list:
+
+### bukkit.yml, spigot.yml, paper.yml, server.properties
+___Gist/pastebin/hastebin links___
+### Other helpful links
+___The more information we receive, the quicker and more effective we can be at finding the solution to the
+issue.___
+
+### Paper build number:
+___This can be found by running `/version` on your server___
```

Comment:
Can we make it explicitly say "Latest is not a version and will not be accepted" here and the other places? Even the source code builds will output a version string with the commit hash, there's really no excuse /rant